tools/add_triage_signature.py

```diff
@@ -1989,6 +1989,47 @@ def _process_ticket(self, url, issue_key):
         self._update_triaging_ticket(report)
 
 
+class HostsIPSignature(Signature):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Host IPv4 addresses:",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+
+        cluster = md["cluster"]
+
+        hosts = []
+        for host in cluster["hosts"]:
+            inventory = json.loads(host["inventory"])
+            hosts.append(
+                OrderedDict(
+                    id=host["id"],
+                    hostname=self._get_hostname(host),
+                    ipv4_addresses=self._get_interfaces(inventory)
+                )
+            )
+
+        report = self._generate_table_for_report(hosts)
+        self._update_triaging_ticket(report)
+
+    def _get_interfaces(self, inventory: dict):
+        ifaces = []
+        for interface in inventory.get("interfaces", []):
+            name = interface.get("name")
+            if not name:
+                continue
+            ifaces.append({
+                "name": name,
+                "ipv4_addresses": interface.get("ipv4_addresses", [])
```

Comment:
We can have one table row per interface, see "Host storage details" signature for how it does it

Reply:
![image](https://user-images.githubusercontent.com/10882062/183878613-b2836eb5-1a4a-49a8-8af8-7038a79ce880.png)
Right now it's not too readable

Reply:
Right now the ipv4_addresses column is not too readable